app/controllers/games_controller.rb

```diff
@@ -1,10 +1,13 @@
 class GamesController < ApplicationController
+  before_action :authenticate_user!
+
   def new
     @game = Game.new
   end
 
   def create
     @game = Game.create(game_params)
+    @game.players.create(game_id: @game.id, user_id: current_user.id, color: [:black, :white].sample)
```

Comment:
Does this look like what we want to do when a new game is created? A Player is created, associated to both the new game and the current_user, and a random color is assigned?